xclim/__init__.py

```diff
@@ -159,4 +167,91 @@ def __build_icclim(mode="warn"):
     return mod
 
 
+def __build_pcc(mode: str = "warn"):
+
+    mapping = dict(
+        # average_length_of_heat_waves,
+        coldest_minimum_temperature=indices.tn_min,
+        cooling_degree_days=indices.cooling_degree_days,
+        # corn_heat_units,
+        date_of_first_fall_frost=utils.wrapped_partial(
+            indices.growing_season_end, thresh="0 degC", mid_date="07-15", window=1
+        ),
+        date_of_last_spring_frost=indices.date_of_last_spring_frost,
+        days_above_32_celsius=utils.wrapped_partial(
+            indices.tx_days_above, thresh="32.0 degC"
+        ),
+        days_above_34_celsius=utils.wrapped_partial(
+            indices.tx_days_above, thresh="34.0 degC"
+        ),
+        dry_days=indices.dry_days,
+        freeze_thaw_cycles=utils.wrapped_partial(
+            indices.daily_freezethaw_cycles,
+            thresh_tasmax="0 degC",
+            thresh_tasmin="-1.0 degC",
+        ),
+        freezing_degree_days=indices.freezing_degree_days,
+        frost_days=indices.frost_days,
+        frost_free_season=indices.maximum_consecutive_frost_free_days,
+        growing_degree_days_base_4_celsius=partial(
```

Comment:
```suggestion
        growing_degree_days_base_4_celsius=wrapped_partial(
```
(And so on for the other lines) `partial` doesn't copy the signature and while our `wrapped_partial` does.

Reply:
Easy fix. Good catch!